docs/extras/building_vtk.rst

```diff
@@ -54,6 +54,9 @@ resulting wheel size.
           -DVTK_OPENGL_HAS_EGL=False \
```

Comment:
Comment for line 49: on Python 3.9 and VTK master I got the following warning from cmake:
```
CMake Warning:
  Manually-specified variables were not used by the project:

    VTK_DATA_EXCLUDE_FROM_ALL
```
Can this be version-dependent? I guess it could only be VTK version dependent.

Reply:
I think that must be a holdover from an older build.  I'll keep this in here since VTK 8.1.2 builds might need it especially given how the windows refuse to close for 9.0.1 on Linux.

Reply:
Yeah, thanks, on 8.1.2 I get this:
```
CMake Warning:
  Manually-specified variables were not used by the project:

    Python3_EXECUTABLE
    VTK_BUILD_DOCUMENTATION
    VTK_BUILD_EXAMPLES
    VTK_BUILD_TESTING
    VTK_MODULE_ENABLE_VTK_PythonInterpreter
    VTK_WHEEL_BUILD
```
This is a disjoint set of switches compared to master so it's all good.

Althought that `Python3_EXECUTABLE` looks a bit suspicious :)